com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/CCWindow/CCWindow.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" ?>
+<operatorModel
+  xmlns="http://www.ibm.com/xmlns/prod/streams/spl/operator" 
+  xmlns:cmn="http://www.ibm.com/xmlns/prod/streams/spl/common" 
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.ibm.com/xmlns/prod/streams/spl/operator operatorModel.xsd">
+  <cppOperatorModel>
+    <context> 
+      <description>Python Windowing Operation</description>
+      <iconUri size="16">../opt/icons/transform_16.gif</iconUri>

```

Comment:
This is an aggregate operator, so it should have aggregate icons.

Reply:
Fixed.